### DISCUSSION TITLE
openssl-3.0.0-alpha17+quic is now available!

### DISCUSSION THREAD

**Discussion by tmshort** [quictls/openssl#33] [Releases]
I have rebased openssl-3.0.0-alpha16+quic onto openssl-3.0.0-alpha17 to create openssl-3.0.0-alpha17+quic, which is now the default branch of the quictls/openssl repository.

This took a little longer due to the latest changes to openssl-3.0.0-alpha16 not being in my repo; also I took a day off. :)

**Comment by tmshort** [comment]
This is expected to be the last alpha release.

**Reply by samsp-msft** [reply]
What's next from OpenSSL - Is there a better handle on the 3.0 timeline?

**Reply by kaduk** [reply]
https://github.com/openssl/openssl/issues/15326 has a checklist of things to be done before beta; the Kanban board at https://app.zenhub.com/workspaces/300-beta-1-573bc8d2d31e1e9a73fff29f/board?repos=7634677 can be used to look at what is being worked on.
